src/main/java/nl/thehyve/ocdu/services/OpenClinicaService.java

```diff
@@ -30,14 +32,14 @@
 @Service
 public class OpenClinicaService {
 
-    SOAPRequestFactory responseFactory = new SOAPRequestFactory();
+    SOAPRequestFactory requestFactory = new SOAPRequestFactory();
```

Comment:
aa... I see you wanted to fixup a commit that was the first one right? Yeah... That indeed requires rebasing on the last commit from master. 
Alternative way would be to change message of the first comit instead of fixing it up and fixup-ing only the middle ones.